### PR TITLE
Trip form a11y

### PR DESCRIPTION
**Pull request by demory**
Adds accessibility support (screenreader / keyboard navigation) to the DateTimeSelector and SettingsSelectorPanel components and subcomponents.

### CODE REVIEW

File: packages/trip-form/src/SubmodeSelector/index.tsx [hunk 1]
```diff
@@ -53,12 +53,18 @@ export default function SubmodeSelector({
...
   return (
-    <S.SubmodeSelector className={className} style={style}>
-      {label && <LabelType>{label}</LabelType>}
+    <S.SubmodeSelector
+      aria-label={label}
+      className={className}
+      role="group"
+      style={style}
+    >
+      {label && <LabelType aria-hidden="true">{label}</LabelType>}
```
Comment (binh-dam-ibigroup) [inline packages/trip-form/src/SubmodeSelector/index.tsx:62]: This label is unused by any input element, so I would convert to a span:
```suggestion
      {label && <LabelType aria-hidden="true" as="span">{label}</LabelType>}
```

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
A few minor changes, and please include a commitizen `fix` commit, so that a new version number can be released.

**Review by binh-dam-ibigroup — APPROVED** [review]
Sorry it took a while to clear all a11y test errors. Should be good to go with our latest changes.

File: packages/trip-form/src/components.story.tsx [hunk 2]
```diff
@@ -158,6 +164,17 @@ export const generalSettingsPanelWithOtp2 = makeStory(GeneralSettingsTemplate, {
   mode: "WALK,BUS,TRAM,SUBWAY",
   otp2: true
 });
+// This error may be raised by us rendering two copies of eagh story.
```
Comment (amy-corson-ibigroup) [inline packages/trip-form/src/components.story.tsx:167]:
```suggestion
// This error may be raised by us rendering two copies of each story.
```

File: packages/trip-form/src/components.story.tsx [hunk 3]
```diff
@@ -120,6 +120,12 @@ export const sliderSelector = makeStory(Core.SliderSelector, {
   style: { display: "inline-block", width: "250px" },
   value: 3
 });
+// This error may be raised by us rendering two copies of eagh story.
```
Comment (amy-corson-ibigroup) [inline packages/trip-form/src/components.story.tsx:123]:
```suggestion
// This error may be raised by us rendering two copies of each story.
```

**Review by amy-corson-ibigroup — APPROVED** [review]
Two small typos, otherwise LGTM!

**Pull request merged**